### PR TITLE
LunaMedia HB Bid Adapter: add video context & outstream handling

### PR DESCRIPTION
**Pull request by lunamedia**
## Type of change

- [x] Feature

## Description of change
Allow publishers to set context:outstream for video demand to get vastXml response.

### CODE REVIEW

**Comment by patmmccann** [comment]
Please see also #6466

File: modules/lunamediahbBidAdapter.js [hunk 1]
```diff
@@ -74,10 +74,13 @@ export const spec = {
       if (mediaType && mediaType[BANNER] && mediaType[BANNER].sizes) {
         placement.sizes = mediaType[BANNER].sizes;
         placement.traffic = BANNER;
-      } else if (mediaType && mediaType[VIDEO] && mediaType[VIDEO].playerSize) {
-        placement.wPlayer = mediaType[VIDEO].playerSize[0];
-        placement.hPlayer = mediaType[VIDEO].playerSize[1];
+      } else if (mediaType && mediaType[VIDEO]) {
+        if (mediaType[VIDEO].playerSize) {
+          placement.wPlayer = mediaType[VIDEO].playerSize[0];
```
Comment (mmoschovas) [inline modules/lunamediahbBidAdapter.js:79]: Prebid nests the playerSize array that is defined in the ad unit, meaning that [640, 480] is converted to [[640,480]]. So currently, you will only be passing an array through placement.wPlayer and placement.hPlayer is undefined and therefore dropped. I believe you are trying to pass the individual number values of the playerSize through their respective properties, so you will need to look into the second array in order to do so.

```
placement.wPlayer = mediaType[VIDEO].playerSize[0][0];
placement.hPlayer = mediaType[VIDEO].playerSize[0][1];
```

**Review by mmoschovas — CHANGES_REQUESTED** [review]

**Comment by ChrisHuie** [comment]
@lunamedia following up on the above reviewer comment for this pr and also issue #6466 

Let me know if I can help on this pr.

**Comment by lunamedia** [comment]
Chris,

Thanks for the note. We will look into this and get back to you!

**Comment by patmmccann** [comment]
please note that the old lunamedia adapter is in violation of #6512 and won't be ported to prebid 5 currently. Is this lunamedia hb your new adapter and the old one is deprecated?

**Comment by lunamedia** [comment]
Hi Patrick,

We are fine to deprecate it but we are transitioning a few clients over.
Can we keep it up for a little bit and confirm once all clients are off it?

**Comment by patmmccann** [comment]
Please also fix #6650 for luna hb on this PR. 

Advise your clients to transition to Luna HB on or before their upgrade to Prebid 5. Your adapter won't stop working unless your clients upgrade to Prebid 5

**Comment by ChrisHuie** [comment]
@lunamedia This pr has been outstanding for a while. Is this still being worked on? We are cleaning out old prs so if I don't hear back on this by Thanksgiving will close and you can reopen when needed.

**Comment by lunamedia** [comment]
Hi Chris,

Adding @Daniel Cohen ***@***.***> and @Mary Drachova
***@***.***> who manage the operation at Luna Media.
We will get back to you as soon as possible

Best,
John

**Pull request closed without merging**